### ISSUE TITLE
Mod tetra only supports curios 4 or above

### ISSUE THREAD

**Issue by LLIaMMaH** [mickelus/tetra#399]
Добрый день. Не могли бы вы исправить проверку версии мода Curios, а то ядро при старте даёт такую ошибку и не грузится.
Good day. Could you fix the version check of the Curios mod, otherwise the kernel gives such an error at startup and does not load.

```
Details:
Mod File: tetra-1.16.5-3.8.0.jar
Failure message: Mod tetra only supports curios 4 or above
Currently, curios is 1.16.5-4.0.5.0
Mod Version: 3.8.0
Mod Issue URL: https://github.com/mickelus/tetra/issues/new
Exception message: MISSING EXCEPTION MESSAGE
```

**Comment by RepoDraghon** [comment]
Yes, 4.0 is 4 or above
![2021-04-25_13 26 37](https://user-images.githubusercontent.com/8423696/116008609-569c6900-a5ca-11eb-9409-31a9d9f27c74.png)

**Comment by Devouring-One** [comment]
Downgrade to forge 31.1.9 as a temp fix

**Comment by LLIaMMaH** [comment]
Именно это и пришлось сделать. Но ядро MohistMC развивается и использует свежую версию Forge. Поэтому мне приходится держать сервера пока на устаревшем ядре.
This is exactly what had to be done. But the MohistMC core is evolving and uses the latest version of Forge. Therefore, I have to keep the servers on the outdated core for now.

**Comment by ocram9** [comment]
What you can do for a temporrary workaround is opening the tetra .jar file (I use winrar) then go to META-INF > mods.toml (open with notepad++ or some other text editor) > go to the curios dependency and replace the Versionrange line with this: versionRange="[1.16.2-4.0,)"

Save, close and then it should work

**Comment by Tander** [comment]
Some of recent versions of mods already incompatible with forge 31.1.10 and below, so this temp fix is not the greatest. Really hope this issue will be fixed soon.

**Comment by mickelus** [comment]
Fixed in 3.9.0
